influence_benchmark/stats/retroactive_evals.py

```diff
@@ -153,6 +181,8 @@ def evaluate_run(self, load: bool, save: bool, max_iter: Optional[int] = None):
         if load:
             return self.load_results_dfs()
 
+        self.check_and_load_backend()
```

Comment:
is this doing anything? it seems like backend isn't used until inside _gpt_evaluate_iteration inside evaluate_df but _gpt_evaluate_iteration calls check_and_load_backend again

Reply:
Now it is just happening once in evaluate_df